core/ir/x86/decode_table.c

```diff
@@ -9251,6 +9289,26 @@ const instr_info_t evex_Wb_extensions[][4] = {
     {OP_vsqrtss, 0xf30f5110, "vsqrtss", Vdq, xx, KE1b, Hdq, Uss, mrm|evex|er|ttt1s, x, END_LIST},
     {OP_vsqrtsd, 0xf20f5140, "vsqrtsd", Vdq, xx, KE1b, Hdq, Wsd, mrm|evex|ttt1s, x, tevexwb[266][3]},
     {OP_vsqrtsd, 0xf20f5150, "vsqrtsd", Vdq, xx, KE1b, Hdq, Usd, mrm|evex|er|ttt1s, x, END_LIST},
+  }, { /* evex_W_ext 267 */
+    {OP_vpdpbusd, 0x66385008, "vpdpbusd", Ve, xx, KEd, He, We, mrm|evex|ttfv, x, END_LIST},
+    {OP_vpdpbusd, 0x66385008, "vpdpbusd", Ve, xx, KEd, He, We, mrm|evex|ttfv, x, END_LIST},
```

Comment:
Why does the manual say `k1` for the write mask, but the type `KEd` can select k1 through k7?

Reply:
I think k1 through k7 are allowed because the binutils tests do use some of these registers. Maybe the manual uses k1 just for illustration - just like it uses zmm1, zmm2 etc.

Reply:
OK, makes sense.